classes/01class/exercises/c01-git03/christina-roperto/c01-git03.txt

```diff
@@ -0,0 +1,21 @@
+#Questions
+
+Create a file c01-git03-1.txt answering the questions below. Include details and commands used.
+
+Did you notice any difference in the size of the repo before and after adding the big file?
+- Yes. The file size is approx. 20MB bigger
+
+What is the reason for this problem?
+- So when the big file is created (10MB), Git also adds the file to its folder(.git). This made it 2x bigger (20MB). When the commit is reverted, the file is removed however Git still stores it in .git folder.
+
+How do you remove something from Git history? Which commands would you use? What are the consequences?
+- You can remove it by typing these commands:
+``` 
+git reflog expire --expire=now --all
```

Comment:
Nice! You can also just drop the commit to solve the problem, using `git rebase -i --root` and adding the option `drop` to the commit you want to remove. Then `git push --force` the branch. It will remove the commit from the remote repository.